dependencies.props

```diff
@@ -72,7 +72,7 @@
   <!-- Package dependency verification/auto-upgrade configuration. -->
   <PropertyGroup>
     <BaseDotNetBuildInfo>build-info/dotnet/</BaseDotNetBuildInfo>
-    <DependencyBranch>master</DependencyBranch>
+    <DependencyBranch>feature/utf8string</DependencyBranch>
```

Comment:
Same comment as in core-setup  there are a bunch of usages of this property below.  Do each of those (projectk-tfs, sni, buildtools, etc) have this branch?